package.json

```diff
@@ -14,6 +14,6 @@
   "main": "lib/index.js",
   "dependencies": {
     "starttls": "^1.0.0",
```

Comment:
Can we lock this down as well? (can we add a .npmrc with save-exact = true )

Reply:
added and locked down